<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QbsProjectManager::Internal::QbsProfilesSettingsWidget</class>
 <widget class="QWidget" name="QbsProjectManager::Internal::QbsProfilesSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="kitLabel">
       <property name="text">
        <string>Kit:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="kitsComboBox"/>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="profileKeyLabel">
       <property name="text">
        <string>Associated profile:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="profileValueLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="propertiesLabel">
     <property name="text">
      <string>Profile properties:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QTreeView" name="propertiesView"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="expandButton">
         <property name="text">
          <string>E&amp;xpand All</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="collapseButton">
         <property name="text">
          <string>&amp;Collapse All</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="editButton">
         <property name="text">
          <string>&amp;Edit...</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
